src/common/components/clusterConfiguration/AdvancedNetworkFields.tsx

```diff
@@ -1,12 +1,13 @@
-import React from 'react';
+import React, { useEffect, useMemo } from 'react';
 import { TextInputTypes } from '@patternfly/react-core';
 import { useFormikContext } from 'formik';
-import { Address6 } from 'ip-address';
+import { InputField } from '../../components/ui';
 import { NetworkConfigurationValues } from '../../types/clusters';
-import { PREFIX_MAX_RESTRICTION } from '../../config';
-import { InputField } from '../ui';
+import { Address6 } from 'ip-address';
+import { PREFIX_MAX_RESTRICTION } from '../../config/constants';
+import { NetworkTypeControlGroup } from '../clusterWizard/networkingSteps/NetworkTypeControlGroup';
 
-const AdvancedNetworkFields: React.FC = () => {
+const AdvancedNetworkFields = () => {
```

Comment:
not sure why the type was removed. It should actually be `React.FC<{}>`. Not a big deal though

Reply:
```suggestion
const AdvancedNetworkFields: React.FC<{}> = () => {
```